### ISSUE TITLE
LEGO Batman crashes with threading, UI elements misplaced

### ISSUE THREAD

**Issue by kuchingneko28** [Hydr8gon/NooDS#105]
https://user-images.githubusercontent.com/73223302/170871089-8e3decb2-a45c-40b3-9ca4-1d80156d422c.MP4

 I get laggy in the loading screen and force close when i press start button

**Comment by Hydr8gon** [comment]
Looks like something is crashing; it's probably related to threading. You could try disabling threaded 2D and 3D in the settings for now. As for the lag, that's just the emulator not reaching full speed on your device.

**Comment by kuchingneko28** [comment]
i disable 2d & 3d threading, now i can play the game but the game run slow and i found this bug
![IMG_20220530_062636](https://user-images.githubusercontent.com/73223302/170893802-ca6a22af-6365-4384-ae19-73612bad3c25.jpg)

Oh yeah btw my device is: 
Xiaomi Redmi note 8
MIUI 12 ( Android 10 )

**Comment by Hydr8gon** [comment]
What's the intended behavior? Are those graphics in the wrong spot? Sorry, I haven't played this game before.

**Comment by kuchingneko28** [comment]
Yes, the yellow bar should be on the top right screen, and the boss health ( tiny heart ) show in a boss fight

**Comment by Hydr8gon** [comment]
4c58e4b fixes the crash, but I'm still not sure about the graphical bug. The game sets those sprites to be drawn there; I assume something is supposed to hide them but I'm not sure what. NO$GBA has this issue as well.

**Comment by kuchingneko28** [comment]
I have try the new updates, i using setting 2d & 3d threading enable and the game running smoothly and nothing crash.

This is result from Lego Batman 1 :
![Screenshot_2022-05-31-23-59-11-471_com hydra noods](https://user-images.githubusercontent.com/73223302/171218455-12957880-45cb-44e1-a81b-82fa2ce78f05.jpg)

This is result from Lego Batman 2 :
![Screenshot_2022-06-01-00-00-42-732_com hydra noods](https://user-images.githubusercontent.com/73223302/171218708-9062acbb-2afd-4f84-a9ce-bf07561bdc45.jpg)

Thank you

**Comment by Hydr8gon** [comment]
I'm keeping this open; I'd still like to look into the graphical issue.
